### DISCUSSION TITLE
What's this doughnut ring mean?

### DISCUSSION THREAD

**Discussion by daterrell** [MikeBishop/MMM-Powerwall#64] [Q&A] [answered]
![Screen Shot 2021-07-29 at 2 43 54 PM](https://user-images.githubusercontent.com/5710124/127569826-06a72bf0-9da4-4111-b31d-60e5d7446fa9.png)

In the self-powered doughnut graph, there's a smaller, gold, inner ring (in my screenshot that inner ring is full).  What does that inner ring represent?

(edited for better screenshot)

**Comment by MikeBishop** [comment, marked as answer]
It's the "solar offset" value -- solar production / (house + car) consumption, full when that reaches or exceeds 100%.
